src/EditorFeatures/CSharpTest/QualifyMemberAccess/QualifyMemberAccessTests.cs

```diff
@@ -1251,6 +1339,30 @@ public async Task QualifyPropertyAccess_InAccessorWithBodyAndExpressionBody2()
 CodeStyleOptions.QualifyPropertyAccess);
         }
 
+        [WorkItem(28509, "https://github.com/dotnet/roslyn/issues/28509")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsQualifyMemberAccess)]
+        public async Task QualifyPropertyAccess_InObjectInitializer()
```

Comment:
❕ This is a collection initializer, not an object initializer (test is correct, but the method name here is wrong)